semcore/wizard/src/Wizard.jsx

```diff
@@ -142,20 +186,72 @@ function Stepper(props) {
 Stepper.enhance = [keyboardFocusEnhance()];
 
 function Content(props) {
-  const { Children, styles } = props;
+  const { Children, styles, uid, step } = props;
   const SContent = Root;
   return sstyled(styles)(
-    <SContent render={Box}>
+    <SContent
+      render={Box}
+      role='tabpanel'
+      aria-labelledby={`${uid}-step-${step}`}
+      id={`${uid}-content-${step}`}
+    >
       <Children />
     </SContent>,
   );
 }
 
+function StepBack(props) {
+  const SStepBack = Root;
+  const { Children, styles, getI18nText, step } = props;
+  const handleClick = React.useCallback(() => {
+    props.onActive?.(props.step - 1);
+  }, [props.step]);
+  return sstyled(styles)(
+    <SStepBack
+      render={Button}
+      use='tertiary'
+      onClick={handleClick}
+      aria-label={getI18nText('backButton', { buttonName: `#${step + 1}` })}
```

Comment:
aria-label should contain the button text instead of the step number
e.g. "Back to Location", "Go to Schedule"
<img width="428" alt="image" src="https://github.com/user-attachments/assets/93bfe613-bea5-4191-9555-a9413d18e6f4">